app/javascript/src/components/Dashboard/Notes/NotesMenu.jsx

```diff
@@ -4,7 +4,7 @@ import { Search, Plus, Settings } from "neetoicons";
 import { Typography } from "neetoui";
 import { MenuBar } from "neetoui/layouts";
 
-const SideMenuBar = () => {
+const NotesMenu = () => {
```

Comment:
https://academy.bigbinary.com/learn-react/pages/structuring-code#:~:text=Avoid%20Unnecessary%20Namespacing

Reply:
I have updated according to the latest reviews. Please review them.

Reply:
>I have updated according to the latest reviews. Please review them.

Moving forward, make sure to add this comment separately out of a reply thread. Meaning as a new separate comment. Otherwise when viewing the PR, the reviewer might miss out your message if this converstaion is resolved.